internal/utils/persistence/src/main/java/org/eclipse/ditto/internal/utils/persistence/mongo/MongoClientWrapper.java

```diff
@@ -25,18 +25,12 @@
 import javax.annotation.concurrent.NotThreadSafe;
 import javax.net.ssl.SSLContext;
 
+import com.mongodb.*;
```

Comment:
I would rather like to prevent wildcard imports, that makes it to "hidden" what actually is used from this package in the import section ..